contracts/modules/staking/StakingPool.sol

```diff
@@ -1269,7 +1271,9 @@ contract StakingPool is IStakingPool, ERC721 {
     rewardsSharesSupply += newRewardsShares;
 
     // transfer nxm from the staker and update the pool deposit balance
-    tokenController.depositStakedNXM(msg.sender, transferAmount, poolId);
+    tokenController.depositStakedNXM(msg.sender, topUpAmount, poolId);
+
+    emit DepositExtended(msg.sender, tokenId, initialTrancheId, newTrancheId, topUpAmount);
```

Comment:
The event `DepositExtended` is only emitted when the initial tranche is not expired. Given that when the initial tranche is expired it performs a `withdraw` and `depositTo`, it looked to me redundant to also emit this event in that case

Reply:
Agreed.